homeassistant/components/freebox/router.py

```diff
@@ -31,6 +31,16 @@
 SCAN_INTERVAL = timedelta(seconds=30)
 
 
+async def get_api(hass: HomeAssistantType, host: str) -> Freepybox:
+    """Get the Freebox API."""
+    freebox_path = Path(hass.helpers.storage.Store(STORAGE_VERSION, STORAGE_KEY).path)
+    freebox_path.mkdir(exist_ok=True)
+
+    token_file = Path(f"{freebox_path}/{slugify(host)}.conf")
+
+    return Freepybox(APP_DESC, token_file, API_VERSION)
```

Comment:
Does this class also support a callback so that HA can store the data in the config entry instead?

Reply:
No it don't, do you have an exemple of integrations that are using callback ?

So I can update the lib for it.

Reply:
https://developers.home-assistant.io/docs/api_lib_auth